docs/upgrading-flatcar.md

```diff
@@ -14,21 +14,58 @@ Occasionally, changes are made to Flatcar Butane configs (those that specify `va
 {:toc}
 
 {% comment %}
+## From Version 1.1.0 to Version 1.2.0-experimental
+
+There are no breaking changes between versions 1.1.0 and 1.2.0-experimental of the `flatcar` configuration specification. Any valid 1.1.0 configuration can be updated to a 1.2.0-experimental configuration by changing the version string in the config.
+
+The following is a list of notable new features.
 
 ## From Version 1.0.0 to Version 1.1.0
 
 There are no breaking changes between versions 1.0.0 and 1.1.0 of the `flatcar` configuration specification. Any valid 1.0.0 configuration can be updated to a 1.1.0 configuration by changing the version string in the config.
 
 The following is a list of notable new features.
 
+### Support Luks discard
+
+Discard; similarly to trim, improves performance for SSDs.
+
+<!-- butane-config -->
+```yaml
+variant: flatcar
+version: 1.1.0
+storage:
+  luks:
+    - name: tang-example
+      device: /dev/sdd
+      discard: true
+```
+
+### Support arbitrary open options on LUKS devices
+
+Allow configuration of arbitrary options to be appended to the LUKS device at bind time.
+
+<!-- butane-config -->
+```yaml
+variant: flatcar
+version: 1.1.0
+storage:
+  luks:
+    - name: tang-example
+      device: /dev/sdd
+#      openOptions: {"optionOne","optionTwo"}
```

Comment:
Hmm, I keep getting an unused field here? idk why but adding the openOptions seems to cause a failing test.. @bgilbert

Reply:
The Butane syntax is `open_options`.  (And then, the list entries should be enclosed in brackets, or on separate lines.)

Reply:
Ah ok thank you for that :)